x-pack/legacy/plugins/code/server/lsp/proxy.ts

```diff
@@ -297,6 +297,6 @@ export class LanguageServerProxy implements ILanguageServerHandler {
 
   public setError(error: any) {
     this.error = error;
-    this.eventEmitter.emit('error', error);
```

Comment:
This is the root cause.  `eventEmitter.emit('error', error)` will throw a exception which stops following codes.
Nodejs doc has never mentioned this.

Reply:
Oh, Fuyao found it here https://nodejs.org/api/events.html#events_error_events